src/styles/index.scss

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
```

Comment:
It's not a good practice to apply `display: flex` and other flex properties directly to the `body` tag. Instead, consider wrapping the content in a container element and apply flexbox styles to it. This provides better encapsulation and reusability of styles.